examples/6-list-activated-methods.py

```diff
@@ -47,5 +43,6 @@ def main():
     except Mollie.API.Error as e:
         return 'API call failed: ' + str(e)
 
+
 if __name__ == '__main__':
-    print(main())
+    print(main())
```

Comment:
https://stackoverflow.com/questions/36043061/automatically-add-newline-on-save-in-pycharm